datasets/swedish_medical_ner/README.md

```diff
@@ -0,0 +1,176 @@
+---
```

Comment:
It looks like this PR contains changes about another dataset that BIOSSES, could you remove these files ?
We can add `swedish_medical_ner` in #2873

Reply:
Thanks @lhoestq ! I have removed the `swedish_medical_ner` folder now.